physlr/physlr.py

```diff
@@ -2195,14 +2195,23 @@ def physlr_path_to_fasta(self):
         num_contigs = 0
         num_bases = 0
 
-        used_seqs = {name[0:-1] for path in paths for name in path if name[-1] != "."}
-
         gaps = "N" * self.args.gap_size
 
         for path in progress(paths):
             if not path:
                 continue
 
+            all_unoriented = True
+            longest_seq = 0
+            for idx, name in enumerate(path):
+                if name[-1] != ".":
+                    all_unoriented = False
+                    break
+                if len(name) > len(path[longest_seq]):
+                    longest_seq = idx
+            if all_unoriented:
+                path[longest_seq] = path[longest_seq][:-1] + "+"
+
             seq = gaps.join(Physlr.get_oriented_sequence(seqs, name)
```

Comment:
Wouldn't we get the same sequences outputted if we check if all the sequences are unoriented, and just move on to the next path in those cases? So just don't place anything here if all are unoriented, then the code below will take care of placing that sequence. 
Just wondering if that's cleaner.

Reply:
Your solution is more correct. It just becomes harder for me to track down the backbone. That's why I did this. Thought I suppose in the end, it doesn't really matter.

Reply:
addressed